.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -79,6 +79,14 @@ class LoadTestsBuilder {
       return baseName
     }
   }
+
+  private static String getContextualDatasetKey(SDK sdk) {
```

Comment:
I'm not sure at this point if we should store that much SDK custom logic in that method. It was a pain either way imo. Maybe we could extract this logic somewhere? WDYT?

Reply:
After an offline discussion we decided to refactor this later: https://issues.apache.org/jira/browse/BEAM-7539